01/app.js

```diff
@@ -0,0 +1,44 @@
+const formEl = document.querySelector("form");
+const usersListEl = document.querySelector(".users-list");
+
+const regexName =
+  /^[a-zA-ZàáâäãåąčćęèéêëėįìíîïłńòóôöõøùúûüųūÿýżźñçčšžÀÁÂÄÃÅĄĆČĖĘÈÉÊËÌÍÎÏĮŁŃÒÓÔÖÕØÙÚÛÜŲŪŸÝŻŹÑßÇŒÆČŠŽ∂ð ,.'-]+$/u;
+
+formEl.addEventListener("submit", handleSubmit);
+
+function handleSubmit(e) {
+  e.preventDefault();
+  const firstName = formEl.elements.firstName;
+  const lastName = formEl.elements.lastName;
+  validateInputEl(firstName);
+  validateInputEl(lastName);
+
+  printData(firstName.value, lastName.value);
+
+  firstName.value = lastName.value = "";
+}
+
+function validateInputEl(inputEl) {
+  const isValid = regexName.test(inputEl.value.trim());
+  if (!isValid) throw new Error("Input is not valid");
```

Comment:
Warto jakoś przejąć ten błąd i pokazać użytkownikowi. Jeśli nie zajrzę do konsoli to nie wiem, że coś nie tak